packages/server/src/util/test-utils/unit.test-generator.ts

```diff
@@ -0,0 +1,34 @@
+import { Org } from '../../api/org/org.model';
+import { ReportSchema } from '../../api/report-schema/report-schema.model';
+import { Unit } from '../../api/unit/unit.model';
+
+export function getUnitsTestData(numUnits: number, org: Org, reportSchemas: ReportSchema[]) {
```

Comment:
We should pull out the creation of musterConfigurations to a generator also (even though they are not serializable entities on their own), thoughts?

Reply:
Yes, and there is lot more that should be extracted. I concentrated on what is needed to get roster and all the related data generated. 
I think this work needs to be part of a new task so that I can move back to completing my primary assignment. 
This is a good discussion and some initial work done with more needed.